cts/lab/ClusterManager.py

```diff
@@ -450,7 +450,7 @@ def isolate_node(self, target, nodes=None):
 
         for node in nodes:
             if node != target:
-                rc = self.rsh(target, self.templates["BreakCommCmd"] % self.key_for_node(node))
+                (rc, _) = self.rsh(target, self.templates["BreakCommCmd"] % self.key_for_node(node))
                 if rc != 0:
                     self.logger.log("Could not break the communication between %s and %s: %d" % (target, node, rc))
                     return None
```

Comment:
Incidentally, SplitBrain is the only experimental test and I believe it does actually work (re-testing that right now).  Maybe we can remove the fact that it's experimental so it'll run by default, and then also get rid of that command line option from ctslab?

Reply:
I think SplitBrain should still be considered experimental because it disables fencing, which would be unpredictable if someone's using a pre-existing configuration. It's a potentially useful test but I think it would need significant changes.

I've got mixed feelings on the experimental designation in general. The concept is useful, but I'd expect that simply not adding it to AllTestClasses would be sufficient for most development purposes. The only real advantage would be the ability to run an experimental test as part of a random sequence of other tests. I suppose it's small enough to keep around.